mlflow/tracking/fluent.py

```diff
@@ -182,10 +207,31 @@ def active_run():
     (parameters, metrics, etc.) through the run returned by ``mlflow.active_run``. In order
     to access such attributes, use the :py:class:`mlflow.tracking.MlflowClient` as follows:
 
-    .. code-block:: py
+    .. code-block:: python
+        :caption: Example
+
+        from pprint import pprint
+        import mlflow
 
         client = mlflow.tracking.MlflowClient()
         data = client.get_run(mlflow.active_run().info.run_id).data
+
+        # Extract only user defined tags; skip System tags starting with "mlflow."
+        tags = {k: v for k, v in data.tags.items() if not k.startswith("mlflow.")}
+
+        # Pretty print the dictionary
+        pprint(data.params)
+        pprint(data.metrics)
+        pprint(tags)
```

Comment:
```suggestion
        from pprint import pprint
        import mlflow

        mlflow.start_run()

        mlflow.log_param("p", 0)
        mlflow.log_metric("m", 1)
        mlflow.set_tag("t", 2)

        client = mlflow.tracking.MlflowClient()
        data = client.get_run(mlflow.active_run().info.run_id).data
        # Extract only user defined tags; skip System tags starting with "mlflow."
        tags = {k: v for k, v in data.tags.items() if not k.startswith("mlflow.")}
        # Pretty print the dictionary
        pprint(data.params)
        pprint(data.metrics)
        pprint(tags)

        mlflow.end_run()
```

Reply:
to make this example runnable?

Reply:
Yes, I can make that. Makes more sense, now come to think of it. 
BTW, since we making some of the examples that are longer runnable, should we use

```
if __name__ == '__main__':
     # Sample code snippet
    ....
```
WDYT?

(To test and to run to fetch output, I have [relevant code in here](https://github.com/dmatrix/mlflow-tests/tree/master/py/mlflow/apis/fluent))